src/utils/_forms.scss

```diff
@@ -29,3 +29,84 @@
     display: none;
   }
 }
+
+// Any form field: input, textarea, ...
+// e.g: @include formField(input);
```

Comment:
a better documentation about how to structure the markup would be much appreciated. ie:
```html
<div class="sui-Form sui-Form-medium sui-Form-success">
  <label>Name</label>
  <input class="sui-Form-input" type="text">
  <span class="sui-Form-success-validation-text">Success message</span>
</div>
```

Reply:
Ok.